templates/config.cli.erb

```diff
@@ -60,7 +98,27 @@ catch
 /subsystem=datasources/jdbc-driver=postgresql:remove
 /subsystem=datasources/jdbc-driver=postgresql:add(driver-module-name=org.postgresql,driver-name=postgresql,driver-xa-datasource-class-name=org.postgresql.xa.PGXADataSource)
 end-try
+<%- end -%><%- # datasource drivers -%> 
+
+<%- else -%><%- # is domain mode -%>
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=driver-name, value=<%= scope['keycloak::datasource_driver'] %>)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=connection-url, value="<%= scope['keycloak::datasource_connection_url'] %>")
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=jndi-name, value=java:jboss/datasources/KeycloakDS)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=user-name, value=<%= scope['keycloak::datasource_username'] %>)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=password, value=<%= scope['keycloak::datasource_password'] %>)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=background-validation, value=true)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=check-valid-connection-sql, value="SELECT 1")
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=background-validation-millis, value=60000)
+/profile=auth-server-clustered/subsystem=datasources/data-source=KeycloakDS:write-attribute(name=flush-strategy, value=IdleConnections)
+try
+/profile=auth-server-clustered/subsystem=datasources/jdbc-driver=postgresql:add(driver-module-name=org.postgresql,driver-name=postgresql,driver-xa-datasource-class-name=org.postgresql.xa.PGXADataSource)
+catch
+/profile=auth-server-clustered/subsystem=datasources/jdbc-driver=postgresql:remove
+/profile=auth-server-clustered/subsystem=datasources/jdbc-driver=postgresql:add(driver-module-name=org.postgresql,driver-name=postgresql,driver-xa-datasource-class-name=org.postgresql.xa.PGXADataSource)
+end-try
 <%- end -%>
+
+<%- if scope['keycloak::operating_mode'] != 'domain' -%>
```

Comment:
Could change this to something like

```
<% if scope['keycloak::truststore'] && if scope['keycloak::operating_mode'] != 'domain' -%>

<% else -%>

<% end -%>
```

Reply:
There is another condition for if is domain...I wonder if instead of multiple blocks for different lines the condition is actually

```
if domain
  profile = '/profile=auth-server-clustered'
else
  profile = ''
end
```

Something like that since that seems to be only difference at first glance but maybe there are other things different.